index.html

```diff
@@ -51,5 +52,183 @@ <h3>LET’S CONNECT</h3>
         </ul>
       </section>
     </div>
+    <section class="grid-container">
+      <div class="grid-item">
+        <div class="image-wrapper">
+          <img src="/images/tonic.png" alt="work-image" />
+        </div>
+
+        <h1>Tonic</h1>
+        <div class="work-details">
+          <p class="first-detail">CANOPY</p>
+          <div class="spacer margin-spacer"></div>
+          <p class="margin-spacer">Back End Dev</p>
+          <div class="spacer margin-spacer"></div>
+          <p class="margin-spacer">2015</p>
+        </div>
+        <h2>
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </h2>
```

Comment:
Please it is  always semantically correct to use the `p` tags for paragraphs instead of the `hx` tag 😄